src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -505,7 +503,7 @@ public synchronized void open() throws SFException, SnowflakeSQLException {
     // overwrite session parameter value with connection parameter value for OOB telemetry in HTAP.
     // Default is disabled.
     if (getBooleanValue(
-            connectionPropertiesMap.get(SFSessionProperty.CLIENT_OUT_OF_BAND_TELEMETRY_ENABLED))) {
+        connectionPropertiesMap.get(SFSessionProperty.CLIENT_OUT_OF_BAND_TELEMETRY_ENABLED))) {
```

Comment:
Can we put debug logs that QCC has been disabled?

Reply:
sure. will do.